frontend/src/metabase/home/components/HomeLayout/HomeLayout.styled.tsx

```diff
@@ -1,71 +0,0 @@
-import styled from "@emotion/styled";
-
-import Button from "metabase/core/components/Button/Button";
-import { hueRotate, lighten } from "metabase/lib/colors";
-import {
-  breakpointMinExtraLarge,
-  breakpointMinLarge,
-  breakpointMinMedium,
-} from "metabase/styled-components/theme";
-
-export const LayoutRoot = styled.div`
-  position: relative;
-  min-height: 100%;
-  padding: 1rem;
-  background-color: var(--mb-color-bg-light);
-
-  ${breakpointMinMedium} {
-    padding: 3rem 4rem;
-  }
-
-  ${breakpointMinLarge} {
-    padding: 4rem 7rem 2rem;
-  }
-
-  ${breakpointMinExtraLarge} {
-    padding: 10rem 15rem 4rem;
-  }
-`;
-
-export const LayoutBody = styled.div`
-  position: relative;
-  margin-top: 2.5rem;
-
-  ${breakpointMinMedium} {
-    margin-top: 4rem;
-  }
-
-  ${breakpointMinLarge} {
-    margin-top: 6rem;
-  }
-`;
-
-export const LayoutIllustration = styled.div<{
-  backgroundImageSrc: string;
-  isDefault: boolean;
-}>`
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  filter: ${({ isDefault }) =>
```

Comment:
from what I can see in the code, `hueRotate` with a color in the `colors` dictionary will always return `0deg`. The code for `hueRotate` is:

```
export const hueRotate = (c: string) => {
  return Color(color(c)).hue() - Color(color(c, originalColors)).hue();
};
```

where (if `c = "brand"`). `colors` is the default param for the 2nd argument:
```
color("brand", [palette = colors]) => colors['brand']
```

and `c = "brand"`, but _originalColors is defined as `{ ...colors }` in `palette.ts`_
```
color("brand", originalColors) => colors['brand']
```

so this call is useless